temporal-normalizer/src/test/java/edu/illinois/cs/cogcomp/temporal/normalizer/tests/TestTemporalChunker.java

```diff
@@ -0,0 +1,354 @@
+        fullFolderName = testFolderURL.getFile();
+        File folder = new File(fullFolderName);
+        File[] listOfFiles = folder.listFiles();
+        DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+        DocumentBuilder builder = factory.newDocumentBuilder();
+        for (File file : listOfFiles) {
+            if (file.isFile()) {
+                String testFile = fullFolderName + "/" + file.getName();
+                byte[] encoded = Files.readAllBytes(Paths.get(testFile));
+                String fileContent = new String(encoded, StandardCharsets.UTF_8);
+                te3inputText.add(fileContent);
+                Document document = builder.parse(new InputSource(new StringReader(fileContent)));
+                Element rootElement = document.getDocumentElement();
+                NodeList nodeList = rootElement.getChildNodes();
+                for (int i = 0; i < nodeList.getLength(); i++) {
+                    Node currentNode = nodeList.item(i);
+                    if (currentNode.getNodeName().equals("TEXT")) {
+                        testText.add(currentNode.getTextContent());
+                    }
+                    if (currentNode.getNodeName().indexOf("DOCID")!=-1) {
+                        docIDs.add(currentNode.getTextContent());
+                    }
+                    if (currentNode.getNodeName().indexOf("DCT")!=-1) {
+                        Node dctNode = currentNode.getChildNodes().item(0);
+                        NamedNodeMap dctAttrs = dctNode.getAttributes();
+                        for (int j = 0; j < dctAttrs.getLength(); j++) {
+                            if (dctAttrs.item(j).getNodeName().equals("value")) {
+                                DCTs.add(dctAttrs.item(j).getNodeValue());
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+    }
+
+    private static final String textFile = "text/AP_20130322";
+
+    @Test
+    public void testPosPipeline() {
+        String input = null;
+        try {
+            input = LineIO.slurp(textFile);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+
+        assertFalse(input.length() == 0);
+
+        AnnotatorService as = null;
+        try {
+            as = PipelineFactory.buildPipeline(ViewNames.POS);
+        } catch (IOException | AnnotatorException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+
+        String norm = input.replaceAll("\\n", " ");
+        norm = StringCleanup.normalizeToUtf8(norm);
+
+
+        try {
+            as.createAnnotatedTextAnnotation("test", "test", norm);
+        } catch (AnnotatorException e) {
+            e.printStackTrace();
+            fail(e.getMessage());
+        }
+    }
+
+    public void testNormalizationWithTrueExtraction() throws Exception {
```

Comment:
if this is running a benchmark test, make it a separate class. Document it and make it easy for user to set the paths for input/output.